README.md

```diff
@@ -1,35 +1,77 @@
 [![Maven Package upon a push](https://github.com/mosip/admin-services/actions/workflows/push-trigger.yml/badge.svg?branch=release-1.3.x)](https://github.com/mosip/admin-services/actions/workflows/push-trigger.yml)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=mosip_admin-services&id=mosip_admin-services&metric=alert_status)](https://sonarcloud.io/dashboard?id=mosip_admin-services)
 
-
 # Admin
 
 ## Overview
 This repository contains the source code MOSIP Admin module. For an overview refer [here](https://docs.mosip.io/1.2.0/modules/administration). The module exposes API endpoints. For a reference front-end UI implementation refer to [Admin UI github repo](https://github.com/mosip/admin-ui/)
 
 Admin module contains following services:
-1. Admin Service
-2. Masterdata Service
-3. Sync Data Service
-4. Hotlist Service
+1. [Admin Service](https://github.com/mosip/admin-services/tree/release-1.3.x/admin/admin-service)
+2. [Masterdata Service](https://github.com/mosip/admin-services/tree/release-1.3.x/admin/kernel-masterdata-service)
+3. [Sync Data Service](https://github.com/mosip/admin-services/tree/release-1.3.x/admin/kernel-syncdata-service)
+4. [Hotlist Service](https://github.com/mosip/admin-services/tree/release-1.3.x/admin/hotlist-service)
 
 ## Databases
 Refer to [SQL scripts](db_scripts).
 
-## Build & run (for developers)
-The project requires JDK 21.0.3
-and mvn version - 3.9.6
-1. Build and install:
-    ```
-    $ cd kernel
-    $ mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true
-    ```
-
-2. Build Docker for a service:
-    ```
-    $ cd <service folder>
-    $ docker build -f Dockerfile
-    ```
+## Build & Run (for developers)
+
+### Prerequisites
+- JDK 21.0.3
+- Maven 3.9.6
+- Docker (latest stable version)
+- Git
+
+### Dependencies
+- Database and configuration files must be set up as per the documentation.
+---
+
+### 1. Build Backend Services
+
+#### a. Clone the repository
+```
+git clone <repo-url>
+cd admin-services
+```
+**Verify**: The repository is cloned and you are in the `admin-services` directory.
+
+#### b. Build all services
+```
+cd admin
+mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgpg.skip=true
+```
+**Verify**: Build completes without errors and JAR files are generated in the `target` directories.
+
+---
+
+### 2. Run Backend Services
+
+#### a. Run using Maven
+```
```

Comment:
- Do the required configuration changes for respective services 
- Mention about config-server need to run before running the service and it should point to local config server